### DISCUSSION TITLE
Creation to an existing storage account

### DISCUSSION THREAD

**Discussion by ghost** [VeeraMS/LogicApp-deployment-with-Secure-Storage#1] [Q&A]
Let me ask you a few questions about this article.
Can this mechanism be implemented for existing storage accounts?
I have a firewall setup on an existing storage account.
I would like to build a StandardLA using that storage account.
I tried writing an ARM template, but it gave me a 403 error.
If it is feasible, I would like to know.

Regards,

**Comment by VeeraMS** [comment]
HI @masakawa527,

Yes you can. You can remove the resources from the template which are already available and just provide the details required in params like storage account name, file share name etc..

Note: You need to create a file share before deploying the Logic App resource either in a template or manually and pass the same in params.

**Comment by ghost** [comment]
Hi @VeeraMS
Thank you very much. In the meantime, I was able to create the resource itself, but the following error was output.

![image](https://user-images.githubusercontent.com/101319740/182590574-8d5d53ba-825b-4977-8f87-a9fb46b10f1c.png)

Is this because Workflow and Standard LA are in the same region?
I could not figure out how to look it up from Storage Explorer.
I will do some more research on my own, but any comments would be appreciated.

**Comment by VeeraMS** [comment]
Hi @masakawa527 ,

this is stating it doesn't have access to storage account from Logic App. Verify the troubleshooting steps specified in this. 

https://techcommunity.microsoft.com/t5/integrations-on-azure-blog/deploying-standard-logic-app-to-storage-account-behind-firewall/ba-p/2626286

**Reply by ghost** [reply]
Thank you. I was able to resolve this one! It was very helpful.
